Error after upgrading ndarray from 0.15 to 0.16

I upgraded `ndarray` from version 0.15 to 0.16 and I'm getting the following error when I try to apply `mod_cholesky_se99` on an instance of `ndarray`:

```
error[E0599]: no method named `mod_cholesky_se99` found for struct `constraints::affine_space::ndarray::ArrayBase` in the current scope
  --> src/constraints/affine_space.rs:54:31
   |
54 |         let res = a_times_a_t.mod_cholesky_se99();
   |                               ^^^^^^^^^^^^^^^^^ method not found in `ArrayBase<OwnedRepr<f64>, Dim<[usize; 2]>>`

```

The same code works with ndarray 0.15.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,9 +16,9 @@ exclude = [
 ]
 
 [dependencies]
-ndarray = { version = "0.15", features = ["approx"] }
+ndarray = { version = "0.16", features = ["approx"] }
 num = "0.4"
-ndarray-rand = "0.14"
+ndarray-rand = "0.15"
 rand = "0.8.3"
 rand_xorshift = "0.3.0"
 

